docs: Add installation guide to mkdocs (#1053)
DSD-DBS/capella-collab-manager: README.md

### Deployment

> The Installation Guide has been moved to the [general documentation](https://capella-collaboration-manager.readthedocs.io/en/latest/installation/).

Reviewer: ```suggestion
You can find the installation guide for deployment in the [general documentation](https://capella-collaboration-manager.readthedocs.io/en/latest/installation/).
```
